packages/admin-ui-docs/src/theme/Footer/styles.module.scss

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+.footer {
+  position: relative;
+  padding: 48px 24px;
+  background: #F8F8F8;
```

Comment:
Could this be a token? Not sure if it makes sense here.

Reply:
Our tokens only work in a object style, I could try to create a object style for this docusaurus component and see if make sense. 🤔